.gitignore

```diff
@@ -0,0 +1,18 @@
+# Created by .ignore support plugin (hsz.mobi)
+### Go template
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+.idea
```

Comment:
`.idea` shouldn't be inside `Dependency directories` section. Maybe add `# Other` or some other comment before it?

Reply:
Fixed!